README.md

```diff
@@ -136,7 +136,7 @@ kustomize build common/istio-1-9-0/istio-install/base | kubectl apply -f -
 
 #### Dex
 
-Dex is an OpenID Connect Identity (OIDC) with multiple authentication backends. In this default installation, it includes a static user named `user`. By default, the user's password is `12341234`. For any production Kubeflow deployment, you should change the default password by following [the relevant section](#change-default-user-password).
+Dex is an OpenID Connect Identity (OIDC) with multiple authentication backends. In this default installation, it includes a static user named `user` with E-mail `user@example.com`. By default, the user's password is `12341234`. For any production Kubeflow deployment, you should change the default password by following [the relevant section](#change-default-user-password).
```

Comment:
I think we can only include email here. Makes this less confusing for users:

Old:

it includes a static user named `user` with E-mail `user@example.com`.

New:
it includes a static user named `user@example.com`.

Reply:
done